data/harohar13.json

```diff
@@ -0,0 +1,11 @@
+{
+  "name": "Harohar",
+  "bio": "Software engineer,writer",
+  "links": [
+    {
+      "name": "Follow me on github",
+      "url": "https://github.com/harohar13",
+      "icon": "github"
```

Comment:
Please use icon name as `FaGithub` 
for more info checkout here https://www.biodrop.io/icons